app/schemas.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
+from datetime import datetime
+from typing import Optional
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: Optional[str] = None
+
+class CityCreate(CityBase):
+    pass
+
+class City(CityBase):
+    id: int
+
+    class Config:
+        orm_mode = True
+
+class TemperatureBase(BaseModel):
+    city_id: int
+    date_time: datetime
+    temperature: float
+
+class TemperatureCreate(BaseModel):
+    city_id: int
+    temperature: float

```

Comment:
In `TemperatureCreate`, consider adding validation to ensure that the `temperature` value is within a realistic range for your application. This can prevent invalid data from being processed.